tests/test_components/test_autograd.py

```diff
@@ -238,14 +250,19 @@ def make_structures(params: anp.ndarray) -> dict[str, td.Structure]:
 
     # static components
     box = td.Box(center=(0, 0, 0), size=(1, 1, 1))
-    med = td.Medium(permittivity=2.0)
+    med = td.Medium(permittivity=3.0)
 
     # Structure with variable .medium
     eps = 1 + anp.abs(vector @ params)
-    conductivity = eps / 10.0
+    sigma = 0.1 * (anp.tanh(vector @ params) + 1)
+
+    permittivity, conductivity = eps, sigma
+    # permittivity, conductivity = eps, 0.0 # only permittivity
+    # permittivity, conductivity = 2.0, sigma # only conductivity
```

Comment:
Should probably make this a parametrized fixture at some point...

Reply:
hm, this is just a remnant from my numerical testing. it's all resolved now. I think I'll just remove these two individual cases.